contact.html

```diff
@@ -90,62 +90,130 @@ <h5>Our Team</h5>
-      <a href="https://www.linkedin.com/in/thealphacod3r/" target="_blank"><img class="git-linkedin"  src="assets/images/linkedin.svg" style="margin: 0 4px;"></a>
-            </div>
-      </div>
         
-      <div class="profile-card">
-        <img src="assets/images/shi2.png" alt="image2" class="profile-icon" />
-        <div class="profile-name">Shilpe Saxena</div>
-        <div class="profile-position">Mentee</div>
-          <div class="links">
-        <a href="https://github.com/shilpe26" target="_blank"><img class="git-linkedin" src="assets/images/github.svg" style="margin: 0 4px;"></a>
-      <a href="https://www.linkedin.com/in/shilpe-saxena-heartly-winner/" target="_blank"><img class="git-linkedin"  src="assets/images/linkedin.svg" style="margin: 0 4px;"></a>
-          </div>
-      </div>
+    <h3 class="title">Project Maintainers</h3>
         
-      <div class="profile-card">
-        <img src="assets/images/v2.png" alt="image3" class="profile-icon" />
-        <div class="profile-name">Varun Verma</div>
-        <div class="profile-position">Mentee</div>
-          <div class="links">
-    <a href="https://github.com/Varun-28" target="_blank"><img class="git-linkedin" src="assets/images/github.svg" style="margin: 0 4px;"></a>
-      <a href="https://www.linkedin.com/in/varun-verma-a-budding-engineer/" target="_blank"><img class="git-linkedin"  src="assets/images/linkedin.svg" style="margin: 0 4px;"></a>
+    <section id="cards-mentor">
+    <div class="mentor-container">
+
+        <div class="card">
+          <div class="imgBx">
+            <img src="assets/images/smaran.png" alt="">
           </div>
-      </div>
-        
-      <div class="profile-card">
-        <img src="assets/images/sonali2.png" alt="image3" class="profile-icon" />
-        <div class="profile-name">Sonali</div>
-        <div class="profile-position">Mentee</div>
-          <div class="links">
-        <a href=" https://github.com/Sonali12920" target="_blank"><img class="git-linkedin" src="assets/images/github.svg" style="margin: 0 4px;"></a>
-      <a href=" https://www.linkedin.com/in/sonali-a989a8192/" target="_blank"><img class="git-linkedin"  src="assets/images/linkedin.svg" style="margin: 0 4px;"></a>
+          <div class="content">
+            <div>
+                <h5>Smaranjit Ghose</h5>
+                <h6>Mentor</h6>
+                <div class="links">
+                <a href="https://github.com/smaranjitghose" target="_blank"><img class="git-linkedin" src="assets/images/github.svg" style="margin: 0 4px;"></a>
+                <a href="https://www.linkedin.com/in/smaranjitghose" target="_blank"><img class="git-linkedin"  src="assets/images/linkedin.svg" style="margin: 0 4px;"></a>
+                </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="imgBx">
+            <img src="assets/images/anush.jfif" alt="">
```

Comment:
Use a folder on Google Drive and collect the images for everyone. Do no use this format for images